1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,11 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {}
+function getBudgets(peopleArray) {
+  let totalBudget = 0;
+  peopleArray.map((person) => totalBudget += person.budget);
+  return totalBudget;
+}
```

Comment:
maybe reduce() method can be interesting for you, s it is suitable for this task too